browser/brave_profile_prefs.cc

```diff
@@ -193,6 +194,11 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Autocomplete in address bar
   registry->RegisterBooleanPref(kAutocompleteEnabled, true);
 
+  // Password leak detection should be disabled
+  registry->SetDefaultPrefValue(
+      password_manager::prefs::kPasswordLeakDetectionEnabled,
+      base::Value(false));
+
```

Comment:
@jumde does this look sufficient?

Reply:
I think this should be grouped with the other overrides above instead adding at the end

Reply:
can be done in a follow up